protocol/src/main/java/org/geysermc/mcprotocollib/network/packet/Packet.java

```diff
@@ -17,4 +17,14 @@ public interface Packet {
     default boolean isPriority() {
         return false;
     }
+
+    /**
+     * Whether the packet is terminal. That means this should be the last packet sent inside a protocol state.
+     * Subsequently, we should disable auto-read when we receive a terminal packet and let the code re-enable auto-read when it's ready to receive more packets.
+     *
+     * @return Whether the packet is terminal.
```

Comment:
```suggestion
     * Returns whether the packet is terminal. If true, this should be the last packet sent inside a protocol state.
     * Subsequently, {@link Session#setAutoRead(boolean)} should be disabled when a terminal packet is received, until the session has switched into a new state and is ready to receive more packets.
     *
     * @return Whether the packet is terminal.
```
probably messed up the link formatting, and requires import